cppclean

```diff
@@ -88,6 +88,19 @@ def main():
                         help='add a header include path; '
                              'specify this multiple times for multiple '
                              'include paths')
+    parser.add_argument('--include-path-system', '-S', action='append',
```

Comment:
Let's just use lower case `-s` and `-n`. I'll merge it afterwards. Thanks for sticking with this!

Reply:
Actually, I can just do that after merging.

Reply:
No problem. Thanks for merging.